src/main/java/com/commercetools/sync/products/helpers/ProductReferenceResolver.java

```diff
@@ -205,20 +205,19 @@ public ProductReferenceResolver(@Nonnull final ProductSyncOptions productSyncOpt
 
         final Map<String, String> categoryOrderHintsMap = new HashMap<>();
         final CategoryOrderHints categoryOrderHints = draftBuilder.getCategoryOrderHints();
-
         return categoryService.fetchMatchingCategoriesByKeys(categoryKeys)
-                              .thenApply(categories ->
-                                  categories.stream().map(category -> {
-                                      if (categoryOrderHints != null) {
-                                          ofNullable(categoryOrderHints.get(category.getKey()))
-                                              .ifPresent(orderHintValue -> categoryOrderHintsMap
-                                                  .put(category.getId(), orderHintValue));
-                                      }
-                                      return category.toReference();
-                                  }).collect(toList()))
-                              .thenApply(categoryReferences -> draftBuilder
-                                  .categories(categoryReferences)
-                                  .categoryOrderHints(CategoryOrderHints.of(categoryOrderHintsMap)));
+                    .thenApply(categories ->
```

Comment:
I could not see any change in here, please do not override the history.

Reply:
Removed the try-catch block here. Sorry for the indentation change in line 209.